app/assets/javascripts/models/ui/reverb_retirement.coffee

```diff
@@ -0,0 +1,19 @@
+#= require util/js.cookies
+
+ns = @edsc.models.ui
+data = @edsc.models.data
+
+ns.ReverbRetirement = do (ko) ->
+  class ReverbRetirement
+    constructor: ->
+
+    returnToReverb: () =>
+      Cookies.set('ReadyForReverbRetirement', 'false', { expires: 7 })
+      $('#reverbRetirementModal').modal('hide')
```

Comment:
I might be wrong but I think the modal will close when the button is clicked. No need to make it close

Reply:
I think you're right - I mean, even if it doesn't disappear, the next line is the redirect back to Reverb which refreshes the page.

I'll delete it and we'll see how it looks when its on SIT.